src/browser/dom/document.zig

```diff
@@ -264,6 +265,10 @@ pub const Document = struct {
         return try TreeWalker.init(root, what_to_show, filter);
     }
 
+    pub fn _createNodeIterator(_: *parser.Document, root: *parser.Node, what_to_show: ?u32, filter: ?NodeIterator.NodeIteratorOpts) !NodeIterator {
+        return try NodeIterator.init(root, what_to_show, filter);
```

Comment:
FYI, you can return without the try. 

The only time you need the explicit try, is when the return type is optional, e.g. `!?NodeIterator` would require a `return try ...`. 

Codebase uses both styles, so either is fine.